### PR TITLE
run coala zuul job

### PR DESCRIPTION
**Pull request by durandom**
let's at least check if our python code has style.

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
       jobs:
-        - "noop"
+        - "thoth-coala"
     gate:
       jobs:
-        - "noop"
+        - "thoth-coala"
     kebechet-auto-gate:
       jobs:
         - "noop"
```
[the remark this answers is not in the document]
Comment (goern) [reply .zuul.yaml:10]: > Oooh. Ok. So we need to make a config change to zuul itself.
> @goern is it possible to point zuul to a wildcard of repos? `aicoe-aiops/*` ?

no.

**Comment by goern** [comment]
is there a specific reason why you want to run coala via zuul? I would like to get you away from any service implementation detail, so I guess all you want it code formatting checks?! we use pre-commit for that now, it seems to be the current best practice with a lot of Python projects...

please add something like https://github.com/thoth-station/template-project/blob/master/.pre-commit-config.yaml to your project template, the CI pipeline will comment (and not fix) any issues in the PR...

**Comment by durandom** [comment]
Do we need to config the CI pipeline then? If I understand this pre-commit thing correct, it only works on the developer machine - or is the same configuration used by the CI pipeline (which comments on the PR)?

**Comment by goern** [comment]
the later, pre-commit (with it's config file in the repository) should be used as a git pre-commit hook locally, and it ist used as part of the CI pipeline that runs on each PR opened

**Comment by harshad16** [comment]
This pull request can be closed, zuul jobs are no longer supported.

**Comment by tumido** [comment]
@harshad16 if zuul jobs are no longer supported, should we remove the `.zuul.yaml` file from this repo then? The other files like `.aicoe-ci.yaml` and `.thoth.yaml` stays right?

**Comment by harshad16** [comment]
@tumido yes, we could remove`.zuul.yaml` file.
`.aicoe-ci.yaml` and `.thoth.yaml` stays

**Pull request closed without merging**